govuk-prototype-kit.config.json

```diff
@@ -1,13 +1,14 @@
 {
   "meta": {
-    "description": "This plugin provides the code to use the GOV.UK Task list pattern in prototypes",
+    "description": "This plugin is deprecated and should be uninstalled. Use the Task list template from ‘Common Templates’ instead",
```

Comment:
**nitpick** People can only uninstall if they run v5, which not every prototype can do at the moment while other plugins update to accomodate the breaking changes. Should we soften the "should be uninstalled" with something like:

"This plugin is deprecated and should only be used if you're running GOV.UK Frontend v4.x. If you're using v5.x and above, it should be uninstalled and use the Task list template from 'Common Templates' instead."

Reply:
We also discussed that you'd need to install the other plugin first and potentially also migrate all your Nunjucks code before uninstalling too. I've not kept track of differences between each Task list component

Tried to meet in the middle with emphasis on _latest_ 'Common Templates' plugin?

>This plugin is deprecated and Task list is now included in GOV.UK Frontend v5.0.0. Uninstall and use the latest ‘Common Templates’ plugin instead, unless GOV.UK Frontend v4 supported is needed.

Reply:
OK done.

Reply:
Sorry I had a typo above if you can swap to ~supported~ support is needed 🤦‍♂️

Reply:
Fixed